user-guide/Cloud_Platform/CloudAdminApp/Controlling_remote_access.md

```diff
@@ -0,0 +1,37 @@
+---
+uid: Controlling_remote_access
+---
+
+# Controlling remote access with the Admin app
+
+When a DMS is cloud-connected, you can use the URL created for you during registration (e.g. ``https://ziine-skyline.on.dataminer.services/``) to access the DataMiner web apps from anywhere.
+
+## Finding the remote access URL
+
+After you have registered, you can find the remote-access URL in the Admin app:
```

Comment:
A better way to use remote access and to get/navigate towards the URL is on the home page, dataminer.services
![image](https://user-images.githubusercontent.com/99406588/154954374-57265152-4425-499a-a1e5-7decb9a3fb7f.png)
Here you have a block for every cloud connected DMS in the selected organization and it lets you navigate to the URLs.

Reply:
Thanks for the feedback! I've added this info in the latest update.